lib/process-request.js

```diff
@@ -6,11 +6,13 @@ const objectToArray = require('./object-to-array');
 
 module.exports = (req, options = {}) => {
   if (options.blacklist) {
-    req.body = removeProperties(req.body, options.blacklist);
+    if (options.blacklist.body) req.body = removeProperties(req.body, options.blacklist.body);
+    if (options.blacklist.headers) req.headers = removeProperties(req.headers, options.blacklist.headers);
```

Comment:
Headers are also case-insensitive... so if you say you want `authorization` removed, I'd expect it to also remove `Authorization` and `authorization`. I think this is good enough for now though! But in future we could use a node version of this object (which is caseless) https://developer.mozilla.org/en-US/docs/Web/API/Headers.